### PR TITLE
Added navbar along with some cleanup

### PR DESCRIPTION
**Pull request by srirag-vuppala**
image
Added hover effects as well

### CODE REVIEW

**Comment by hhokari** [comment]
Looks good, maybe consider implementing a hamburger for responsiveness

**Comment by srirag-vuppala** [comment]
> Looks good, maybe consider implementing a hamburger for responsiveness

Will do! I'm sure it'll be a priority for after we do our MVP

File: node_modules/.bin/acorn [hunk 1]
```diff
@@ -1 +0,0 @@
-../acorn/bin/acorn
```
Comment (hhokari) [inline node_modules/.bin/acorn:1]: Why is there almost 2000 removals of node modules from outside of the src folder

Comment (srirag-vuppala) [reply node_modules/.bin/acorn:1]: I had to remove it for clarity! It was unneeded so thought this might have been the best approach

**Pull request merged**